easybuild/base/fancylogger.py

```diff
@@ -277,7 +277,7 @@ def log_method(self, msg):
 
     # method definition as it is in logging, can't change this
     # pylint: disable=unused-argument
-    def makeRecord(self, name, level, pathname, lineno, msg, args, excinfo, func=None, extra=None):
+    def makeRecord(self, name, level, pathname, lineno, msg, args, excinfo, func=None, extra=None, sinfo=None):
```

Comment:
some documentation on the arguments wouldn't go amiss

Reply:
Yeah, I agree, but the Python documentation itself is rather light on this too...

This is only used internally in `fancylogger`, I'd rather not spend too much time on this.

Reply:
See https://docs.python.org/2/library/logging.html#logging.Logger.makeRecord